tests/http2-disabled/test.yaml

```diff
@@ -0,0 +1,28 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  min-version: 7.0.0
+
+# disables checksum verification
+args:
+  - -k none --set stream.midstream=true
+
+checks:
+
+  # Check that there is one file event with content range.
+  - filter:
+      count: 1
+      match:
+        event_type: http
```

Comment:
can we add something here to show the alert happens on time (so at the moment of the non-switch)? I guess pcap_cnt or flow bytes/pkts might be available.